### PR TITLE
Started cacheAsBitmap tests

### PR DESCRIPTION
**Pull request by staff0rd**
Currently failing due to #3681

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
btw, "isWebGLSupported" is changed to "withGL"

File: test/extras/cacheAsBitmap.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+            expect(stage.width).to.be.equal(20);
+        });
+
+        it('on webgl without cacheAsBitmap', withGL(function ()
+        {
+            const renderer = new PIXI.WebGLRenderer(150, 150);
+            const stage = getScene(renderer, false);
+
+            expect(stage.width).to.be.equal(20);
```
Comment (englercj) [inline test/extras/cacheAsBitmap.js]: Do we have a way to test that what we expect to be drawn is actually drawn?

**Comment by bigtimebuddy** [comment]
@staff0rd could you fix the linting error?

**Comment by bigtimebuddy** [comment]
I would love to get these tests in. @ivanpopelyshev could you look at why the CanvasRenderer test for cacheAsBitmap is failing? Looks like some math problem considering the pivot point.

**Comment by bigtimebuddy** [comment]
I'm going to close this because of inactivity. Need a solution for handling `cacheAsBitmap` with pivot.

**Pull request closed without merging**